numpy-stubs/__init__.pyi

```diff
@@ -450,6 +451,11 @@ def array(
     subok: bool = ...,
     ndmin: int = ...,
 ) -> ndarray: ...
+def asarray(
```

Comment:
Is it possible to add some tests for this?

Reply:
Done.